lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/BroadcastReceiver.cs

```diff
@@ -79,6 +76,17 @@ public sealed class BroadcastReceiver<T> : IBroadcastReceiver<T>
         public string OperatorName { get; private set; }
 
         /// <summary>
+        /// Ensure all parent and children nodes in the topology are registered with teh Name Service.
+        /// </summary>
+        public void WaitForRegistration()
```

Comment:
Do these methods need to be `public`? They are purely internal to the group communications implementation, right?